package.json

```diff
@@ -94,14 +94,15 @@
     "body-parser": "^1.19.0",
     "chokidar": "^3.0.2",
     "cjson": "^0.3.1",
-    "cli-color": "^2.0.2",
+    "cli-color": "2.0.1",
     "cli-table": "0.3.11",
     "commander": "^4.0.1",
     "configstore": "^5.0.1",
     "cors": "^2.8.5",
     "cross-env": "^5.1.3",
     "cross-spawn": "^7.0.1",
     "csv-parse": "^5.0.4",
+    "es5-ext": "0.10.53",
```

Comment:
Should this be an [override](https://stackoverflow.com/questions/58930922/google-cloud-functions-custom-build-step) instead / in addition?

Reply:
The override functionality isn't working for me (I suspect `npm-shrinkwrap.json` has to do with it)